docs/features/docker_compose.md

```diff
@@ -51,3 +51,10 @@ if err != nil {
 return nil
 ```
 
+If your compose stack creates volumes you may need to remove them with the `--volumes` option to prevent dangling volumes accumulating on test runners.
```

Comment:
Do you thing is a flag we should always pass to the `down` method?

Reply:
Yes, it would probably be a good default for testcontainers because it's very unlikely that any users are relying on these volumes persisting between compose runs. For test usage we just want everything to be cleared up.

Reply:
Would you mind adding it please? 🙏

Reply:
Sure. I will send a PR soon.

Reply:
Please see https://github.com/testcontainers/testcontainers-go/pull/381